### PR TITLE
adjusted sidebar style and pulled out datatable into component

### PR DESCRIPTION
**Pull request by scramb**
## Description

Rebase from #349 

Sidebar now select multiple correctly and shows buttons for basic actions like share download and delete - if selected multiple download is disabled for now.
## Related Issue

N/A
## Motivation and Context

N/A
## How Has This Been Tested?

-

## Screenshots (if appropriate):
2018-11-26-131436_1917x900_scrot
2018-11-26-131423_1921x900_scrot

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] Fix Sidebar Headline

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -33,7 +33,7 @@ pipeline:
       - php occ log:manage --level 0
       - php occ config:list
       - php occ a:e oauth2
-      - php occ oauth2:add-client Phoenix Cxfj9F9ZZWQbQZps1E1M0BszMz6OOFq3lxjSuc8Uh4HLEYb9KIfyRMmgY5ibXXrU 930C6aA0U1VhM03IfNiheR2EwSzRi4hRSpcNqIhhbpeSGU6h38xssVfNcGP0sSwQ http://phoenix:8300
+      - php occ oauth2:add-client Phoenix Cxfj9F9ZZWQbQZps1E1M0BszMz6OOFq3lxjSuc8Uh4HLEYb9KIfyRMmgY5ibXXrU 930C6aA0U1VhM03IfNiheR2EwSzRi4hRSpcNqIhhbpeSGU6h38xssVfNcGP0sSwQ http://phoenix:8300/#/login
```
Comment (DeepDiver1975) [inline .drone.yml]: this was already in #355

File: package.json [hunk 2]
```diff
@@ -38,7 +38,7 @@
     "eslint-plugin-node": "^8.0.0",
     "eslint-plugin-promise": "^4.0.1",
     "eslint-plugin-standard": "^4.0.0",
-    "eslint-plugin-vue": "^5.0.0-beta.3",
+    "eslint-plugin-vue": "^5.0.0-beta.5",
```
Comment (DeepDiver1975) [inline package.json]: #321

File: src/services/auth.js [hunk 3]
```diff
@@ -10,6 +10,7 @@ export function initVueAuthenticate (config) {
           clientId: config.clientId,
           url: config.apiUrl,
           authorizationEndpoint: config.authUrl,
+          redirectUri: window.location.href,
```
Comment (DeepDiver1975) [inline src/services/auth.js]: #355

File: package.json [hunk 4]
```diff
@@ -53,7 +53,7 @@
     "style-loader": "^0.23.1",
     "vue-loader": "15.4.2",
     "vue-template-compiler": "^2.5.16",
-    "webpack": "^4.26.0",
+    "webpack": "^4.26.1",
```
Comment (DeepDiver1975) [inline package.json]: #354

**Comment by scramb** [comment]
got it - im too stupid to merge closed and rebased to #360

**Pull request closed without merging**